maxmin/min2/.keep

```diff
@@ -0,0 +1,14 @@
+package main
```

Comment:
min2 program doesn't work atm - you have to rename `.keep` into smth like `main.go`